### PR TITLE
[mercadopago] fix: add missing properties to interfaces

### PR DESCRIPTION
**Pull request by danieldspx**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## mercadopago ([<kbd>unpkg</kbd>](https://unpkg.com/browse/mercadopago@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'mercadopago'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against f3981d911dca5ca178a419002b0f0c08836ae735
</p>

**Comment by Vractos** [comment]
@danieldspx Could you fix the interface of `CreateCustomerPayload` as well? It's missing the `email` parameter. So it would be like that:

```
export interface CreateCustomerPayload {
  /**Email do cliente */
  email:string
  /** Nome do cliente. */
  first_name?: string;
  /** Sobrenome do cliente. */
  last_name?: string;
  /** Telefone do cliente. */
  phone?: Omit<Phone, 'extension'>;
  /** Informações sobre a identificação do cliente. */
  identification?: Identification;
  /** Endereço por defeito do cliente. */
  default_address?: string;
  /** Informação sobre o endereço padrão do cliente. */
  address?: SimpleAddressId;
  /** Data (ISO_8601) de registo do cliente. */
  date_registered?: string;
  /** Descrição do cliente. */
  description?: string;
  /** Metadata do cliente */
  metadata?: any;
  /** Cartão padrão do cliente. */
  default_card?: string;
}

```

**Comment by danieldspx** [comment]
> o

Sure, look at how the documentation is garbage: https://www.mercadopago.com.br/developers/pt/reference/customers/_customers/post/ in the example they use the email property, but the do not define it in the **Request Params** section :cry:

File: types/mercadopago/models/customers/create-payload.model.d.ts [hunk 1]
```diff
@@ -3,6 +3,8 @@ import { Identification } from '../../shared/payer-identification';
 import { Phone } from '../../shared/phone';
 
 export interface CreateCustomerPayload {
+  /**Email do cliente */
+  email:string
```
Comment (Vractos) [inline types/mercadopago/models/customers/create-payload.model.d.ts]: Sorry, I forgot to set the parameter to optional. Can you change to `email?:`, please?

**Review by Vractos — CHANGES_REQUESTED** [review]

File: types/mercadopago/models/customers/create-payload.model.d.ts (hunk 1 above)
Comment (danieldspx) [reply types/mercadopago/models/customers/create-payload.model.d.ts]: done

**Review by Vractos — APPROVED** [review]

**Review by Vractos — CHANGES_REQUESTED** [review]
@danieldspx Could you fix the interface of `SimpleAddress` (mercadopago/shared/address.d.ts) as well? The `street_number` parameter's declared as a number, but it's string

**Review by Vractos — APPROVED** [review]

**Review by amcasey — APPROVED** [review]

**Comment by danieldspx** [comment]
Ready to merge

**Comment by danieldspx** [comment]
Ready to merge

**Comment by nternouski** [comment]
Still missing "metadata" attribute in CreatePreferencePayload, can you update too? Thanks!

**Comment by danieldspx** [comment]
> Still missing "metadata" attribute in CreatePreferencePayload, can you update too? Thanks!

You can contribute. I would love to review a PR.

**Pull request merged**